docs/style-guide.md

```diff
@@ -199,3 +201,22 @@ We could enforce a camel case convention on these strings, such as the following
 ##### Decision
 
 - Defer the decision on the syntax for identifiers to the other specification when possible.
+
+## Element Ordering
+
+This section concerns the order in which containers store elements. 
+
+### General Guidelines
+
+:star2: ECMA-402 spec must provide a deterministic order for the contents of all containers. This order should be lexicographic except in cases wherein there is a clearly preferable semantic ordering. :star2:
+  
+For an example of when *not* to use lexicographic order, consider an array holding days of the week. This array should store those days in the order `["monday", "tuesday", "wednesday", ...]` instead of the lexicographic `["friday", "monday", "saturday", "sunday", ...]` order.
```

Comment:
Choose something that is actually in the spec:

```suggestion
For an example of when *not* to use lexicographic order, consider an array holding days of the week. This array should store those days in the order `["years", "months", "weeks", "days"]` instead of the lexicographic `["days", "months", "weeks", "years"]` order.
```

Reply:
```suggestion
For an example of when *not* to use lexicographic order, consider an array holding calendar time scale units. This array could use descending magnitude as `["years", "months", "weeks", "days"]` or its reverse ascending magnitude, but should not use the lexicographic `["days", "months", "weeks", "years"]` order.
```

Please also provide at least one positive example.